.env.example

```
@@ -17,6 +17,7 @@ BUSINESS_MSSQL_SOURCE_SA_PASSWORD=ChangeMeDevOnly_123
 SAFEQUERY_APP_NAME="SafeQuery API"
 SAFEQUERY_APP_POSTGRES_URL=postgresql://safequery:change-me-for-shared-environments@app-postgres:5432/safequery
 SAFEQUERY_ENVIRONMENT=development
+SAFEQUERY_DEV_AUTH_ENABLED=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SAFEQUERY_DEV_AUTH_ENABLED key should go before the SAFEQUERY_ENVIRONMENT key

(UnorderedKey)